OxfordBodleian/Juel-Jensen/BDLaethe38.xml

```diff
@@ -136,51 +412,171 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                <desc>Fine and regular handwriting, slightly deteriorating towards the end (e.g. <locus from="76v" to="78v"/>).
+                                </desc>
+                                <seg type="rubrication">
+                                    Holy names and the names of the individuals involved in the production of the book, both in the colophon and in the supplication formulas throughout the
+                                    manuscript. Several groups of lines on the incipit page of <ref target="#i2"/> and of the opening sections for each weekday, sometimes alternating with black lines. 
+                                    The words <foreign xml:lang="gez">ሰላም፡</foreign> in the colophon (<ref target="#i1"/>) and 
+                                    <foreign xml:lang="gez">ሰላም፡ ለኪ፡</foreign> in <ref target="#i4 #i5"/>. 
+                                    Elements of the punctuation, including text dividers, and numerals, including the quire marks.
+                                </seg>
+                                <list type="abbreviations">
+                                    <item>
+                                        <abbr>ለዓ፡ ዓ፡ አ፡</abbr> for <expan>ለዓለመ፡ ዓለም፡ አሜን፡</expan> (<locus target="#12ra"/>)
+                                    </item>
+                                    <item>
+                                        <abbr>አ፡</abbr> for <expan>አሜን፡</expan> (<locus target="#60va"/>)
+                                    </item>
+                                </list>
                             </handNote>
                         </handDesc>
                         
                        <!-- <decoDesc>
-                         
                         </decoDesc>-->
                         
-                     <!--   <bindingDesc>
-                            <binding xml:id="binding">
-                                <decoNote xml:id="b1"></decoNote>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <desc type="Unclear"></desc>
+                                    <locus target="#3r"/>
+                                    <q xml:lang="gez">
+                                        ዕፁበ፡ ግብረ፡ ወመድምመ፡ ኢርኢኩ፡ ወኢሰማዕኩ፡ ከዳዊት፡ ከሰሎሞን፡ ከዛቲ፡ ዘመን፡ ንጉሥ፡ ለስደት፡ ንጉሥ፡ መሸኃዳር፡ ኢርኢኩ፡ ወኢሰማዕኩ፡ 
+                                        ገዳማተ፡ ትግሬ፡ ወኢትዮጵያ፡ ብክያ፡ ወላህዋ፡ እምሥራቅ፡ እስከ፡ ምዕራብ፡ መንግሥት፡ እናጥፋ፡ አሉ፡ መሳ<add place="above">ፍ</add>ንት፡ 
+                                        ወመኳንንት፡ ዕፁበ፡ ግብረ፡ ወምድ<cb n="b"/>ምመ፡ ኢርኢኩ፡ ወኢሰማዕኩ፡ ንጉሥ፡ ለስደት፡ ንጉሥ፡ መሸኃዳር፡ ኢርኢኩ፡ ወኢሰማዕኩ፡ 
+                                        ገዳማተ፡ ትግሬ፡ ወኢትዮጵያ፡ ብክያ፡ ወላህዋ፡ 
+                                    </q>
+                                    <note>The note is written in Gǝʿǝz with some words and expressions in Amharic. The end of the note is barely legible due to a water stain.</note>
```

Comment:
the content of note would better be include in `<desc>`, then there is no need for the note element.